common/src/main/java/earth/terrarium/olympus/client/components/dropdown/DropdownAlignment.java

```diff
@@ -0,0 +1,37 @@
+package earth.terrarium.olympus.client.components.dropdown;
+
+import net.minecraft.client.gui.components.AbstractWidget;
+import org.joml.Vector2i;
+
+public enum DropdownAlignment {
```

Comment:
Move this to a common location and rename it to OverlayAlignment, I'd like to use it for the color picker